### ISSUE TITLE
RSS Feed for Blog

### ISSUE THREAD

**Issue by thomasbnt** [WarenGonzaga/bmc-updates-bot#1]
Is better to integrate that with a webhook or direcly the bot to sending in the channel ?

Reference : https://github.com/WarenGonzaga/bmc-updates-bot/commit/3e10248e306abc675a36df337e0687c5ff4a1944#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28fR14-R49

**Comment by warengonzaga** [comment]
@thomasbnt it is better if we can add an option for BMC Crew which channel they want to appear the updates by using commands.

**Comment by thomasbnt** [comment]
@WarenGonzaga BMC Crew can edit that with simply Webhook app 😄 
We can move the app to another text channel

**Comment by warengonzaga** [comment]
> @WarenGonzaga BMC Crew can edit that with simply Webhook app 😄
> We can move the app to another text channel

what webhook app is this?

**Comment by thomasbnt** [comment]
Here, you are the management of the WebHook url @WarenGonzaga https://github.com/WarenGonzaga/bmc-updates-bot/blob/8b5e2df518550b24de8d1ee5c824940520b9e36b/app.js#L16

**Comment by warengonzaga** [comment]
> Here, you are the management of the WebHook url @WarenGonzaga
> 
> https://github.com/WarenGonzaga/bmc-updates-bot/blob/8b5e2df518550b24de8d1ee5c824940520b9e36b/app.js#L16

You nailed it!

**Comment by thomasbnt** [comment]
Bug with the RSS feed of BMC blog. Need contact BMC Crew for more help

**Comment by warengonzaga** [comment]
> Bug with the RSS feed of BMC blog. Need contact BMC Crew for more help

Is this already solved?

**Comment by thomasbnt** [comment]
@WarenGonzaga nope, test that and check if I created a issue with my code, I didn't seen any problem :c

**Comment by warengonzaga** [comment]
> @WarenGonzaga nope, test that and check if I created a issue with my code, I didn't seen any problem :c

Got it.. will check this one
